src/geogenalg/attributes.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) 2025 National Land Survey of Finland (Maanmittauslaitos)
+#
+#  This file is part of geogen-algorithms.
+#
+#  This source code is licensed under the MIT license found in the
+#  LICENSE file in the root directory of this source tree.
+
+import geopandas as gpd
+
+
+def inherit_attributes(
+    source_gdf: gpd.GeoDataFrame, target_gdf: gpd.GeoDataFrame
+) -> gpd.GeoDataFrame:
+    """Copy attributes from the source_gdf to the geometries in target_gdf.
+
+    Args:
+    ----
+        source_gdf: GeoDataFrame containing polygons with original attributes
+        target_gdf: GeoDataFrame with new geometries that need inherited attributes
+
+    Returns:
+    -------
+        A GeoDataFrame containing the target geometries with inherited attributes from
```

Comment:
I think it would be good idea to clarify this a bit. Something like "result dataframe contains attributes from the source dataframe but no attributes from the target dataframe".